README.md

```diff
@@ -150,6 +150,56 @@ co(function * () {
 
 ```
 
+### Shared Secret Example
+
+Sometimes it is desirable that the sender can choose the amount and generate the
+condition without communicating with the recipient. This is an example of a
+payment using the Shared Secret Protocol (SSP) which implements this type of
+flow.
+
+```js
+'use strict'
+
+const co = require('co')
+const ILP = require('.')
+const FiveBellsLedgerPlugin = require('ilp-plugin-bells')
+
+const sender = ILP.createSender({
+  _plugin: FiveBellsLedgerPlugin,
+  account: 'https://localhost/ledger/accounts/alice',
+  password: 'alice'
+})
+
+const receiver = ILP.createReceiver({
+  _plugin: FiveBellsLedgerPlugin,
+  account: 'https://localhost/ledger/accounts/bob',
+  password: 'bobbob'
+})
+
+co(function * () {
+  yield receiver.listen()
+  receiver.on('incoming', (transfer, fulfillment) => {
+    console.log('received transfer:', transfer)
+    console.log('fulfilled transfer hold with fulfillment:', fulfillment)
+  })
+
+  const secret = receiver.generateSharedSecret()
+  console.log('secret:', secret)
+
+  const request = sender.createRequest(Object.assign({}, secret, {
```

Comment:
Yeah, the reason I implemented it this way is because I didn't want to create a second quoting method.

You can think of it like this: In SSP, both the sender and the receiver can create requests (because the secret is shared). (So the sender doesn't have to ask the receiver for a request, it can just generate one.) IPR is basically the same as SSP, except the receiver does not share the secret, so only the receiver can generate requests.

Not sure if "request" is the right name, it's more of a "payment description" or something. You called it "packet", but I don't think it's the same as an ILP packet, so I'd prefer calling it something else.